.github/workflows/selfdrive_tests.yaml

```diff
@@ -91,6 +91,34 @@ jobs:
         path: .ci_cache/scons_cache
         key: scons-${{ env.CACHE_COMMIT_DATE }}-${{ github.sha }}
 
+  build_arm64:
+    name: build all (arm64)
+    runs-on: buildjet-4vcpu-ubuntu-2204-arm
```

Comment:
probably only need 2 vcpu, which is the same as what's on the github machines. Can you also make this a composite since it's repeated?

Reply:
what do you mean by composite?

Reply:
similar to .github/workflows/setup, like a macro to be used elsewhere